e2e/Import.e2e.ts

```diff
@@ -0,0 +1,39 @@
+import { setElectronDialogHandler } from 'testcafe-browser-provider-electron';
+import { assertNoConsoleErrors, cleanup, getPageUrl, pageUrl } from './helpers';
+import { Home } from './pages';
+
+fixture`Import`
+  .page(pageUrl)
+  .beforeEach(Home.clickImportButton)
+  .afterEach(assertNoConsoleErrors)
+  .afterEach(cleanup);
+
+test('should be on the import network route', async t => {
+  await t.expect(getPageUrl()).match(/network_import/);
+});
+
+test('when the user aborts the file dialog, nothing should happen', async t => {
```

Comment:
I'm having issues with this test. I can't get Testcafe to trigger the file dialog... Any idea what I'm doing wrong?